youtube_dl/YoutubeDL.py

```diff
@@ -848,6 +850,22 @@ def iter_playlistitems(format):
                 random.shuffle(entries)
 
             for i, entry in enumerate(entries, 1):
+                if self.params.get('playlistprompt', False):
+                    if entry['title']:
+                        skipnextvid = False
+                        while True:
+                            answer = compat_input('Do you want to download video %s/%s: "%s"? (y/n) ' %
+                                                  (i, n_entries, entry['title'].encode('ascii', 'ignore')))
```

Comment:
Even the Standard-Youtube Functions don't manage to show anything usefull for this Unicode Sign in the Video Title:
> Destination: Wolbachia - The Bacteria That Could Change Everything -DCDhTLikAwo.f137.mp4

Also trying to copy&Paste this specific Unicode Char ends in two question marks.
This is the sign we're daling with:
http://www.fileformat.info/info/unicode/char/1f30f/index.htm